kubetest/e2e.go

```diff
@@ -241,9 +241,6 @@ func run(deploy deployer, o options) error {
 
 func listNodes(dump string) error {
 	b, err := output(exec.Command("./cluster/kubectl.sh", "--match-server-version=false", "get", "nodes", "-oyaml"))
-	if verbose {
```

Comment:
I'm not sure I'm following.
if err != nil, then this finished with an error, so this doesn't contain nodes (but just some error). not sure if it's worth logging anything.